src/frontend/src/lib/components/convert/ConvertForm.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import ConvertAmount from '$lib/components/convert/ConvertAmount.svelte';
+	import Button from '$lib/components/ui/Button.svelte';
+	import ButtonGroup from '$lib/components/ui/ButtonGroup.svelte';
+	import ContentWithToolbar from '$lib/components/ui/ContentWithToolbar.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { OptionAmount } from '$lib/types/send';
+
+	export let sendAmount: OptionAmount;
+	export let receiveAmount: number | undefined;
+	export let totalFee: bigint | undefined;
+	export let disabled: boolean;
+	export let insufficientFunds: boolean;
+	export let insufficientFundsForFee: boolean;
+
+	const dispatch = createEventDispatcher();
+</script>
+
+<form on:submit={() => dispatch('icNext')} method="POST">
```

Comment:
We have other components that wraps a form around everything? This is needed here?

Reply:
I think I saw it somewhere in the send flow, but you're right, there is no need in it. Removed it.